.github/workflows/pytest-workflow.yml

```diff
@@ -95,7 +95,7 @@ jobs:
         if: failure()
         uses: actions/upload-artifact@v2
         with:
-          name: logs-${{ matrix.tags }}-${{ matrix.profile }}-${{ matrix.nxf_version }}
+          name: logs-${{ matrix.profile }}-${{ matrix.nxf_version }}
```

Comment:
Before merging note this was necessary, because the tags have a `/` in them. I created #571 to address this later.